src/JsonForm/Fields/FormTextInput.tsx

```diff
@@ -44,7 +44,7 @@ export const FormTextInput = (props: FormTextInputProps) => (
 				...field,
 				onChange: undefined,
 				...props,
-				error: (form.errors[name] && form.touched[name]) || props.error,
+				error: form.errors[name] ? true : false || props.error,
```

Comment:
```suggestion
				error: !!form.errors[name] || props.error,
```

Can you test this instead

Reply:
it worked fixed it